x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/index_view_logic.ts

```diff
@@ -203,6 +207,20 @@ export const IndexViewLogic = kea<MakeLogicType<IndexViewValues, IndexViewAction
       () => [selectors.indexData],
       (index) => (isConnectorViewIndex(index) ? index.connector.id : null),
     ],
+    hasAdvancedFilteringFeature: [
+      () => [selectors.connector],
+      (connector?: Connector) =>
+        connector?.features ? connector.features[FeatureName.FILTERING_ADVANCED_CONFIG] : false,
+    ],
+    hasBasicFilteringFeature: [
+      () => [selectors.connector],
+      (connector?: Connector) =>
+        connector?.features ? connector.features[FeatureName.FILTERING_RULES] : false,
+    ],
+    hasFilteringFeature: [
+      () => [selectors.hasAdvancedFilteringFeature, selectors.hasBasicFilteringFeature],
+      (advancedFeature: boolean, basicFeature: boolean) => advancedFeature && basicFeature,
```

Comment:
What's the purpose of the && here?  Should it be ||?

Reply:
Yup! Good catch